Documentation/format/index.md

```diff
@@ -139,11 +139,11 @@ The `#names` field determines the number indexed label names, followed by the to
 └─────────────────────────────────────────────────┘
 ```
 
-The sequence of label index sections is finalized by an offset table pointing to the beginning of each label index section for a given set of label names.
+The sequence of label index sections is finalized by an [offset table](#offset-table) pointing to the beginning of each label index section for a given set of label names.
```

Comment:
The wording needs to be fixed here. "is finalized by" makes it sound like the section itself ends with an offset table which isn't the case.